### ISSUE TITLE
Incorrect version number  with SDWebImage:master branch 

### ISSUE THREAD

**Issue by HamGuy** [SDWebImage/SDWebImage#3412]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

It seems like the latest version(5.13.4) was bumped in a fork branch: [dreampiggy/SDWebImage](https://github.com/dreampiggy/SDWebImage), but  not sync to the **SDWebImage:master** branch.

**Comment by dreampiggy** [comment]
Solved. Try re-pull the latest version.
